module4/src/com/epam/logic/TrainWorker.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.logic;
+
+import com.epam.entity.Train;
+
+
+public class TrainWorker {
+
+    private static int counter = 0;
+    private static Train yourTrain;
```

Comment:
these variables can be local in method - getTrainByNumber. And name - yourTrain really bad. Just train or something like this.